src/handle-data.js

```diff
@@ -15,12 +15,20 @@ export function getColumnOrder({ qColumnOrder, qDimensionInfo, qMeasureInfo }) {
   return qColumnOrder?.length === columnsLength ? qColumnOrder : [...Array(columnsLength).keys()];
 }
 
-export function getColumnInfo(qHyperCube, colIndex) {
+export function getColumnInfo(qHyperCube, colIndex, layout, translator) {
   const { qDimensionInfo, qMeasureInfo } = qHyperCube;
   const numDims = qDimensionInfo.length;
   const isDim = colIndex < numDims;
   const info = isDim ? qDimensionInfo[colIndex] : qMeasureInfo[colIndex - numDims];
   const isHidden = info.qError?.qErrorCode === 7005;
+  const totalInfo = isDim
+    ? colIndex === 0
+      ? layout.totals.label !== undefined
+        ? layout.totals.label
+        : translator.get('Object.Table.Totals')
```

Comment:
this will be translated later
```suggestion
        : 'Object.Table.Totals'
```